x-pack/plugins/upgrade_assistant/server/lib/telemetry/usage_collector.ts

```diff
@@ -136,6 +138,7 @@ export function registerUpgradeAssistantUsageCollector({
           cluster: { type: 'long' },
           indices: { type: 'long' },
           overview: { type: 'long' },
+          kibana: { type: 'long' },
```

Comment:
Would you mind documenting the telemetry fields? We are trying to enforce documentation of, at least, the new collected fields.

You can do so by adding _meta.description like

```ts
{ type: 'long', _meta: { description: 'number of times users opened the Kibana deprecation page (I guess?) ' } }
```

Thanks!

Reply:
👍 Thanks for pointing this out @pgayvallet! I've added descriptions for most of the telemetry UA is collecting in https://github.com/elastic/kibana/pull/97159/commits/67bd3093641ca13221f43387477c78901017b4e7. I left the reindex actions as-is for now, as we have future plans to change the UI here and I'm not sure yet how it will affect telemetry. I've made a note to follow up. Can you take another look when you get a chance?